pandas/core/generic.py

```diff
@@ -5548,53 +5548,74 @@ def clip(self, lower=None, upper=None, axis=None, inplace=False,
         """
         Trim values at input threshold(s).
 
+        Truncates values below and above specified thresholds.
+        Thresholds can be singular values or array like, and in
+        the latter case the truncation is performed element-wise
+        in the specified axis.
+
         Parameters
         ----------
         lower : float or array_like, default None
+            Minimum threshold value. All values below this
+            threshold will be set to it.
         upper : float or array_like, default None
+            Maximum threshold value. All values above this
+            threshold will be set to it.
         axis : int or string axis name, optional
             Align object with lower and upper along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
+        args : iterable, optional
+            Arguments to pass to the function.
+        kwargs : mapping, optional
+            Keyworded arguments to pass to the function.
+
+        See Also
+        --------
+        clip_lower : Clips values below specified threshold(s).
+        clip_upper : Clips values above specified threshold(s).
 
         Returns
         -------
         clipped : Series
 
         Examples
         --------
+        >>> data = {'0': [9, -3, 0, -1, 5], '1': [-2, -7, 6, 8, -5]}
+        >>> df = pd.DataFrame(data)
         >>> df
-                  0         1
-        0  0.335232 -1.256177
-        1 -1.367855  0.746646
-        2  0.027753 -1.176076
-        3  0.230930 -0.679613
-        4  1.261967  0.570967
-
-        >>> df.clip(-1.0, 0.5)
-                  0         1
-        0  0.335232 -1.000000
-        1 -1.000000  0.500000
-        2  0.027753 -1.000000
-        3  0.230930 -0.679613
-        4  0.500000  0.500000
-
+            0 	1
+        0 	9 	-2
+        1 	-3 	-7
+        2 	0 	6
+        3 	-1 	8
+        4 	5 	-5
```

Comment:
alignment of the printed output seems strange. Should align right for each individual column